weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterBar.tsx

```diff
@@ -170,6 +174,12 @@ export const FilterBar = ({
     value: 'id',
     label: 'Call ID',
   });
+  if (isWandbAdmin) {
+    (options[0] as GroupedOption).options.push({
+      value: 'feedback.[*].trigger_ref',
```

Comment:
So this filters to all calls that are monitored? In the event that we add other trigger types, this will over-select. Might want to just add a note